src/common/utils/gpo.h

```diff
@@ -115,6 +118,128 @@ namespace powertoys_gpo {
         }
     }
 
+    inline std::string getPolicyListValue(const std::wstring& registry_list_enabled_value, const std::wstring& registry_list_path, const std::wstring& registry_list_value_name)
+    {
+        // This function returns the value of an entry of an policy list. The user scope is only checked if the list is not enabled for the machine (registry_list_enabled_value) to not mix them. If the no value or no list is found in the registry we return an empty string.
+
+        HKEY key{};
+        DWORD value = 0xFFFFFFFE;
+        DWORD valueSize = sizeof(value);
+
+        bool machine_list_found = false;
+        if (auto res = RegOpenKeyExW(POLICIES_SCOPE_MACHINE, POLICIES_PATH.c_str(), 0, KEY_READ, &key); res == ERROR_SUCCESS)
+        {
+            // If the path was found in the machine, we need to check if the value for the policy exists.
+            auto resValue = RegQueryValueExW(key, registry_list_enabled_value.c_str(), nullptr, nullptr, reinterpret_cast<LPBYTE>(&value), &valueSize);
+            RegCloseKey(key);
+
+            if (resValue != ERROR_SUCCESS)
```

Comment:
Simplify: 

```
if (resValue == ERROR_SUCCESS) {
if (value == 1) {
machine_list_found = true
}
}

otherwise machine_list_found is set to false on init.